<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Receptionist</class>
 <widget class="QMainWindow" name="Receptionist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Pic_Label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>201</width>
      <height>251</height>
     </rect>
    </property>
    <property name="text">
     <string>Place Picture Herel</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>171</width>
      <height>257</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="ReserveHall">
       <property name="text">
        <string>Reserve Hall</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ReserveTable">
       <property name="text">
        <string>Reserve Table</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ReserveRoom">
       <property name="text">
        <string>Reserve Room</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ReserveVehicle">
       <property name="text">
        <string>Reserve Vehicle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CheckRes">
       <property name="text">
        <string>Check Avilable Resources</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="updateOrDelete">
       <property name="text">
        <string>Update or Delete Reservation</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CheckOut">
       <property name="text">
        <string>Perform Check Out</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="LogOut">
       <property name="text">
        <string>Log Out</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>389</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
